express-notes-api/index.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+const fs = require('fs');
+
+app.get('/api/notes', (req, res) => {
+  const newArr = [];
+  for (const x in data.notes) {
+    newArr.push(data.notes[x]);
+  }
+  res.json(newArr);
+});
+app.get('/api/notes/:id', (req, res) => {
+  const id = Number.isInteger(req.params.id);
```

Comment:
This will make your `id` be `true` or `false` and not a number. `Number.isInteger` should be used inside the `if` statement condition. Make sure to test your code before submitting pull requests.